### PR TITLE
Add initial version

### PR DESCRIPTION
**Pull request by AntiD2ta**

### CODE REVIEW

File: metrics.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    for blob in data.get('result', {}).get('data', {}).get('json', {}).get('data', []):
+        requested_at = blob.get('requested_at', 0)
+        time_difference = (current_time - requested_at) / 3600  # Convert seconds to hours
+
+        metrics['blob_index'].set(blob.get('blob_index', 0))
+        metrics['reference_block_number'].set(blob.get('reference_block_number', 0))
+        metrics['batch_id'].set(blob.get('batch_id', 0))
+        metrics['confirmation_block_number'].set(blob.get('confirmation_block_number', 0))
+        metrics['time_difference'].set(time_difference)
+        # Update additional metrics here
```
Comment (adriantpaez) [inline metrics.py:39]: Is this update performed for each blob? If so, why not use just the latest blob, specifically the one with the most recent `requested_at`?

File: metrics.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    Update the Prometheus metrics with the latest data from the API.
+
+    Args:
+    metrics (dict): The dictionary of Prometheus Gauges.
+    data (dict): The data fetched from the API.
+    """
+    current_time = time.time()
+    for blob in data.get('result', {}).get('data', {}).get('json', {}).get('data', []):
+        requested_at = blob.get('requested_at', 0)
+        time_difference = (current_time - requested_at) / 3600  # Convert seconds to hours
```
Comment (adriantpaez) [inline metrics.py:32]: I recommend not converting the `time_difference` to hours. This conversion can be done with PromQL in the query without reducing the precision of the data source.

**Review by adriantpaez — CHANGES_REQUESTED** [review]

File: metrics.py (hunk 1 above)
Comment (AntiD2ta) [reply metrics.py:39]: The idea is to expose all the blob data, not only the `requested_at` related information. It is in the interest of the data consumer to get what is helpful to them from the dataset (i.e., on Grafana, you use what you want; maybe the rest is useful for a different context).

Do you have any concerns with the current solution?

File: metrics.py (hunk 2 above)
Comment (AntiD2ta) [reply metrics.py]: Good recommendation. Done [87ef4e8](https://github.com/NethermindEth/eigenda-blob-scrapper/pull/1/commits/87ef4e86bc30b520cd6b6d93985a98fe477f2415)

File: metrics.py (hunk 1 above)
Comment (adriantpaez) [reply metrics.py]: Following an internal discussion, we've decided to:

- Remove the `blob_index` metric
- Sort blobs by their `requested_at` timestamp
- Refrain from adding blobs with outdated `requested_at` timestamps

**Review by adriantpaez — APPROVED** [review]

**Pull request merged**